src/ctest.ts

```diff
@@ -285,24 +285,15 @@ export class CTestDriver implements vscode.Disposable {
             }
 
             if (!testExplorer) {
-                log.info(localize('no.tests.found', 'No tests found'));
+                log.info(localize('test.explorer.not.enabled', 'ctest integration disabled. Please see the `cmake.ctest.testExplorerIntegrationEnabled` setting.'));
                 return -1;
             }
 
-            if (!this.ws.config.ctestAllowParallelJobs) {
-                const tests = this.testItemCollectionToArray(testExplorer.items);
-                const run = testExplorer.createTestRun(new vscode.TestRunRequest());
-                const ctestArgs = await this.getCTestArgs(driver, customizedTask, testPreset);
-                const returnCode = await this.runCTestHelper(tests, run, driver, undefined, ctestArgs, undefined, customizedTask, consumer);
-                run.end();
-                return returnCode;
-            } else {
-                const retc = await this.runCTestDirectly(driver, customizedTask, testPreset, consumer);
-
-                // not sure if direct comparison can be made to replace reloadTests with refreshTests
-                await this.refreshTests(driver);
-                return retc;
-            }
+            const tests = this.testItemCollectionToArray(testExplorer.items);
+            const run = testExplorer.createTestRun(new vscode.TestRunRequest());
+            const ctestArgs = await this.getCTestArgs(driver, customizedTask, testPreset);
+            const returnCode = await this.runCTestHelper(tests, run, driver, undefined, ctestArgs, undefined, customizedTask, consumer);
+            return returnCode;
```

Comment:
@gcampbell-msft i do not understand why this is modified. Should we assume here that `ctestAllowParallelJobs` is always true?